hedera-node/hedera-file-service-impl/src/main/resources/genesis/simpleFeesSchedules.json

```diff
@@ -257,6 +256,78 @@
           "name": "TokenCancelAirdrop",
           "baseFee": 9000000,
           "extras": []
+        },
+        {
+          "name": "TokenAssociateToAccount",
+          "baseFee": 499000000,
+          "extras": [
+            { "name": "TOKEN_TYPES", "includedCount": 0 }
```

Comment:
Same here, this extra doesn't belong here right?

Reply:
That's a good question.  [The HIP](https://hips.hedera.com/hip/hip-1261) lists several transactions, including this one, as using the TOKEN_TYPES extra. However, I don't understand what it is used for.   The original handlers don't use  anything equivalent.